### ISSUE TITLE
Error: No articles to display

### ISSUE THREAD

**Issue by alexmelo** [GavickPro/News-Show-Pro-GK5#147]
News Show Pro GK 5 installed in Joomla 3.2.3 and activate the module by selecting a category joomla, articles are not displayed.

What to do?

Screen Home:
![tela2](https://f.cloud.github.com/assets/315018/2518413/f1a0da04-b463-11e3-875d-d6dd667e084a.jpg)

Screen Data Source Module:
![tela](https://f.cloud.github.com/assets/315018/2518390/9e2d5c12-b463-11e3-8454-ec847f872d54.jpg)

**Comment by martinwhyte** [comment]
hello,

After installing News Show Pro GK5, i outputs "Error: No articles to display" i've set the offset starting ranging from -12 to +12 and still the same error. im trying to use it for Easy Blog. But i noticed when i run the site on XAMPP on my local host, News Show Pro GK5 works fine but once i upload it on a live server, it reads "Error: No articles to display"! pls how do i sort this out?

**Comment by ListOrSwap** [comment]
Hello! Thank you anyone in advance who can help me!

So.. I am having the same problem, except.. I have two NSPro modules..

The first one displays K2 items, just fine.
The second.., however, does not.. It just has the error: No Article to Display...

I have changed everything I can think of.. Even made new K2 Item, New K2 Category, New NSPro module and still, same thing.. 
Then I tried switching the item category to the one that works.. and it displays..

I am at my wits end....

Thank you again to anyone that can help me!

Kyle, with a smile :)
